changes-set.txt

```diff
@@ -27,6 +27,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 7-Oct-22 exmoeu2   exmoeub     biconditional form of exmoeu
```

Comment:
~eubi was moved from a mathbox to main, so it should be documented here...

Reply:
Will do, @avekens. Although this change cannot break any proof that was relying on the pre-change version, so to me the signaling is not necessary.

Reply:
@avekens : can you approve that latest commit ?